### DISCUSSION TITLE
Can't find stylesheet to import. Angular library

### DISCUSSION THREAD

**Discussion by victorcamargos** [nolimits4web/swiper#4898] [Q&A]
With the version 7 of swiper and the use of style exports I am not able to make the import of the css/scss work for Angular library.
It works fine for an Angular app project.
Angular library uses ng-packagr as a builder but I could not find a solution for the import, setting it to ~swiper/css/bundle gets rid of the error but styles are not being applied.
Does anyone have any clue on how to make it work on a library project?

ERROR: Can't find stylesheet to import.
  ╷
1 │ @import 'swiper/css/bundle';
  │         ^^^^^^^^^^^^^^^^^^^

**Comment by ha-akelius** [comment]
I did a workaround by 
`@import '~swiper/swiper.scss';`
